test/over_react_redux/connect_flux_test.dart

```diff
@@ -793,7 +742,7 @@ main() {
       });
 
       expect(
-          logs.first,
+          logs[1],
```

Comment:
Because of the `Cannot find native JavaScript type (SyntheticMouseEvent) for type check` warning (caused by the removal of the `@anonymous` annotations from `SyntheticEvent` js interop classes in `react-dart`), the expected warning appears second in the logs.

Reply:
Also this is the only non-formatting change in this file.

Reply:
Updating the test in this way makes it depend on that error existing, but that won't always be the case. Can we instead update it to ignore that error, so that it won't start failing again if that error goes away?
```dart
      final logs = await recordConsoleLogsAsync(() async {
        click(fluxButton);
        await Future(() {});
      });
      logs.removeWhere((log) => log.contains('Cannot find native JavaScript type');

      expect(
          logs.first,
```

Reply:
Ohh yep, good call. I added that!